lib/include/openamp/rpmsg_virtio.h

```diff
@@ -53,6 +53,7 @@ struct rpmsg_virtio_shm_pool {
 struct rpmsg_virtio_config {
 	uint32_t h2r_buf_size;
 	uint32_t r2h_buf_size;
+	bool split_shpool;
```

Comment:
The field description is missing  above. Please also update the commit subject according to the new config name

Reply:
Done!